packages/vscode-extension/package.json

```diff
@@ -167,19 +167,19 @@
         {
           "id": "teamsfx-help-and-feedback",
           "name": "Help and feedback",
-          "when": "fx-extension.isTeamsFx && !fx-extension.canUpgradeV3 && !fx-extension.isV3Enabled",
+          "when": "fx-extension.isTeamsFx && !fx-extension.canUpgradeV3",
           "visibility": "visible"
         },
         {
           "id": "teamsfx-help",
```

Comment:
Why not remove the section if not used anymore?

Reply:
I think it's possible to be reused again when we need to publish a new prerelease version in future.

Reply:
Do you mean that this feature is just a temporary change and will be reverted in later prerelease?